src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -2262,7 +2260,7 @@ def _download(
         progress_bar : bool, optional
             Display a progress bar using ``tqdm`` when ``True``.
```

Comment:
I'd propose this as starting with "Whether to" is our standard format for a Boolean parameter:

Whether to display a progress bar using ``tqdm``. The default is ``False``. Because showing
download progress is helpful, you might want to set this parameter to ``True``.

Maybe explain why they wouldn't want to change it to ``True``, which would help them understand why the default is ``False``?

Reply:
See:

https://github.com/ansys/pymapdl/pull/2053#discussion_r1194910595